modules/apps/frontend-theme/frontend-theme-admin/src/css/_custom.scss

```diff
@@ -47,3 +47,7 @@ body {
 		margin-top: 20px;
 	}
 }
+
+.card-page-item-asset {
```

Comment:
Is this the only place this happens? Do all cards have this class?

Also... what about the card views outside of `admin`? I'm not sure about others, but Document Library has a public view outside of `admin`...

Reply:
We could apply it s a common pattern but and we will do it in clay, for the moment this sheet-lg with cards inside appears only in admin side so my POV the temporal line is fone.
We will remove it as soon as clay be updated with the general change

Reply:
fair enough

Reply:
we can start with `admin` since the bug was found in there, then we can extend the fix to other modules (or wait for clay update that fix them all).